src/experiments/webgpuaudio/test/test_processor.js

```diff
@@ -0,0 +1,28 @@
+import GPUProcessor from "../gpu-processor.js";
+import IRHelper from "../ir-helper.js"
+
+class TestProcessor {
+
+    constructor() {}
+    testConvolution = async() => {
```

Comment:
I was surprised that this syntax works, but the formal ES6 notation would be:
```js
constructor() {}

async testConvolution() {
...
}
```

Reply:
Done!